.travis.yml

```diff
@@ -22,6 +22,10 @@ script:
      - if [ "$BUILD_NET_TOKIO" == "1" ]; then RUSTFLAGS="-C link-dead-code" cargo build --verbose; fi
      - if [ "$BUILD_NET_TOKIO" != "1" ]; then RUSTFLAGS="-C link-dead-code" cargo build --verbose -p lightning; fi
      - rm -f target/debug/lightning-* # Make sure we drop old test binaries
+     # Run clippy on Rust 1.39.0
+     - if [ "$(rustup show | grep default | grep 1.39.0)" != "" ]; then
+           rustup component add clippy &&
+           cargo clippy; fi
```

Comment:
Good if you want to add a note in CONTRIBUTING.md #Coding Conventions precising the ones enforced by clippy and thus what a developer should care about would be nice. A reference to accurate clippy docs works too.

Reply:
added instructions to CONTRIBUTING.MD#Coding Conventions!